### PR TITLE
feat(secmaster): add data source metric results

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:
add data source metric results

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```
add data source metric results
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST="./huaweicloud/services/acceptance/secmaster" TESTARGS="-run TestAccDataSourceMetricResults_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/secmaster -v -run TestAccDataSourceMetricResults_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceMetricResults_basic
=== PAUSE TestAccDataSourceMetricResults_basic
=== CONT  TestAccDataSourceMetricResults_basic
--- PASS: TestAccDataSourceMetricResults_basic (17.71s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/secmaster 17.772s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

### CODE REVIEW

File: docs/data-sources/secmaster_metric_results.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* `labels` - The statistical labels of the metric.
+  The value in the label corresponds to the value in a piece of `data_row` one-to-one.
+
+* `data_rows` - All statistical data of the metric.
+  The [data_rows](#metric_results_data_rows) structure is documented below.
+
+<a name="metric_results_data_rows"></a>
+The `data_rows` block supports:
+
+* `data_row` - a piece of data in the metric results.
```
Comment (houpeng80) [inline docs/data-sources/secmaster_metric_results.md]:
```suggestion
* `data_row` - A piece of data in the metric results.
```

File: huaweicloud/services/secmaster/data_source_huaweicloud_secmaster_metric_results.go [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"timespan": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"cache": {
+				Type:     schema.TypeString,
+				Optional: true,
```
Comment (profoundwu) [inline huaweicloud/services/secmaster/data_source_huaweicloud_secmaster_metric_results.go]: Please add a description for the field.

**Pull request merged**